Sming/Services/SpifFS/spiffs_config.h

```diff
@@ -136,6 +136,15 @@
 #define SPIFFS_USE_MAGIC                (0)
```

Comment:
> The code related to SPIFFS_USE_MAGIC doesn't work as expected as pointed in #1660, but I decided to leave it.

I guess the default value here should be (1) and not (0).

Reply:
Also starting to use MAGIC will make all existing SPIFFS to be treated as invalid and will be formatted. Such a change needs to be communicated with care to the developers otherwise their apps will stop working as expected :(.